README.md

```diff
@@ -1,15 +1,34 @@
 # Jackson Datatype Money
 
-[Jackson](http://jackson.codehaus.org) module (jar)
-to support JSON serialization and deserialization of
-[javax.money](https://github.com/JavaMoney/jsr354-api) data types.
-
 [![Build Status](https://img.shields.io/travis/zalando/jackson-datatype-money.svg)](https://travis-ci.org/zalando/jackson-datatype-money)
 [![Coverage Status](https://img.shields.io/coveralls/zalando/jackson-datatype-money.svg)](https://coveralls.io/r/zalando/jackson-datatype-money)
 [![Release](https://img.shields.io/github/release/zalando/jackson-datatype-money.svg)](https://github.com/zalando/jackson-datatype-money/releases)
 [![Maven Central](https://img.shields.io/maven-central/v/org.zalando/jackson-datatype-money.svg)](https://maven-badges.herokuapp.com/maven-central/org.zalando/jackson-datatype-money)
 
-## Dependency
+Jackson Datatype Money is a [Jackson](https://github.com/codehaus/jackson) module (jar) to support JSON serialization and deserialization of [Java Money](https://github.com/JavaMoney/jsr354-api) data types. It fills a niche, in that it connects the Java Money and Jackson libraries so that they work seamlessly together, without requiring additional developer effort. In doing so, it aims to perform a small but repetitive task—once and for all.
+
+## Features
+Jackson Datatype Money:
+-  enables you to express monetary amounts in JSON
+-  can be used in a REST API
+-  offers customizable serialization by localelo
+-  allows you to implement RESTful API endpoints that format monetary amounts based on the Accept-Language header 
+- can be used in a desktop application
```

Comment:
I know I said this and it's true, but I don't know whether it's important enough to be highlighted here.